rero_ils/modules/migrations/api.py

```
@@ -61,7 +61,7 @@ class Index:
         """Migration Index configuration."""
 
         name = "migrations-20240909"
-        settings = {"number_of_shards": 2, "number_of_replicas": 2}
+        settings = {"number_of_shards": 1, "number_of_replicas": 1}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)